api/config/packages/sentry.yaml

```diff
@@ -3,6 +3,18 @@ sentry:
     register_error_listener: false # Disables the ErrorListener to avoid duplicated log in sentry
     options:
         environment: '%env(SENTRY_ENVIRONMENT)%'
+        integrations:
+            - 'Sentry\Integration\IgnoreErrorsIntegration'
+
+services:
+    Sentry\Integration\IgnoreErrorsIntegration:
+        arguments:
+            $options:
+                ignore_exceptions:
+                    - Symfony\Component\HttpKernel\Exception\NotFoundHttpException
+                    - Symfony\Component\Routing\Exception\ResourceNotFoundException
+                    - Symfony\Component\Security\Core\Exception\AccessDeniedException
+                    - Symfony\Component\HttpKernel\Exception\HttpException
```

Comment:
From meeting discussion: Consider using a ValidationException class to trigger 422 errors. This would avoid ignoring exception, if anyone in the future uses HttpException for non 4xx errors